common/src/mailbox_api.rs

```diff
@@ -210,6 +215,28 @@ impl GetLdevCertResp {
     pub const DATA_MAX_SIZE: usize = 1024;
 }
 
+/// QUOTE_PCRS input arguments
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct QuotePcrsReq {
+    pub hdr: MailboxReqHeader,
+    pub nonce: [u8; 32],
+}
+
+pub type PcrValue = [u8; 48];
+
+/// QUOTE_PCRS output
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct QuotePcrsResp {
+    pub hdr: MailboxRespHeader,
+    /// The PCR values
+    pub pcrs: [PcrValue; 32],
```

Comment:
Might be useful to include the nonce in the response, just to allow the response to be cryptographically verifiable on its own.

Reply:
Since that's not how the spec defines it, should I just alter the spec with this PR or should the spec changes go into a different one?

Reply:
You can update the spec in the same PR